### ISSUE TITLE
Expo Router template fails on start with `Error: createCSSVariable expected string, got: undefined`

### ISSUE THREAD

**Issue by keith-kurak** [tamagui/tamagui#2849]
### Current Behavior

I run `npm create tamagui@latest`, choose the Expo Router template, run `yarn start`, get the following error and the process terminates:

```
keith@andknuckles dailydots % yarn start
Starting project at /Users/keith/Github/dailydots
Starting Metro Bundler
warning: Bundler cache is empty, rebuilding (this may take a minute)

    ➡ [tamagui] built config and components (493ms):

        Config     ./.tamagui/tamagui.config.cjs
        Components ./.tamagui/tamagui-components.config.cjs
        
  tamagui: skipping /Users/keith/Github/dailydots/.tamagui/tamagui.config.cjs tamagui.dev/docs/intro/errors#warning-001
  tamagui: skipping /Users/keith/Github/dailydots/.tamagui/tamagui-components.config.cjs tamagui.dev/docs/intro/errors#warning-001

/Users/keith/Github/dailydots/node_modules/@tamagui/metro-plugin/node_modules/@tamagui/web/src/createVariable.ts:108
      throw new Error(`createCSSVariable expected string, got: ${nameProp}`)
            ^
Error: createCSSVariable expected string, got: undefined
    at createCSSVariable (/Users/keith/Github/dailydots/node_modules/@tamagui/metro-plugin/node_modules/@tamagui/web/src/createVariable.ts:108:13)
    at variableToCSS (/Users/keith/Github/dailydots/node_modules/@tamagui/metro-plugin/node_modules/@tamagui/web/src/helpers/registerCSSVariable.ts:10:63)
    at /Users/keith/Github/dailydots/node_modules/@tamagui/metro-plugin/node_modules/@tamagui/web/src/createTamagui.ts:129:29
    at createTamagui (/Users/keith/Github/dailydots/node_modules/@tamagui/metro-plugin/node_modules/@tamagui/web/src/createTamagui.ts:194:3)
    at loadTamagui (/Users/keith/Github/dailydots/node_modules/@tamagui/metro-plugin/node_modules/@tamagui/static/src/extractor/loadTamagui.ts:61:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at Object.load [as loadTamagui] (/Users/keith/Github/dailydots/node_modules/@tamagui/metro-plugin/node_modules/@tamagui/static/src/extractor/createExtractor.ts:132:29)
```

The error looks the same as this: https://github.com/tamagui/tamagui/issues/2530, except it happens without modifications to the template. I tried to adopt the workarounds suggested at the time but they did not work in this situation. Some other users are reporting the same behavior in that closed issue thread as of a few days ago.

### Expected Behavior

`yarn start` doesn't error/terminate, and I can then run the app on mobile or web.

### Tamagui Version

```markdown
1.108.3
```

### Platform (Web, iOS, Android)

```markdown
All
```

### Reproduction

```markdown
https://github.com/keith-kurak/create-tamagui-expo-router/tree/main
```

### System Info

```markdown
System:
    OS: macOS 14.2.1
    CPU: (8) arm64 Apple M1 Pro
    Memory: 60.05 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.15.1 - ~/.volta/tools/image/node/20.15.1/bin/node
    Yarn: 4.0.2 - ~/.volta/tools/image/yarn/1.22.17/bin/yarn
    npm: 10.7.0 - ~/.volta/tools/image/node/20.15.1/bin/npm
    Watchman: 2024.07.15.00 - /opt/homebrew/bin/watchman
  Browsers:
    Chrome: 127.0.6533.100
    Safari: 17.2.1
  npmPackages:
    @babel/core: ^7.24.6 => 7.24.6
    @expo/metro-config: ~0.18.4 => 0.18.4
    @expo/metro-runtime: ~3.2.1 => 3.2.1
    @react-navigation/native: ^6.1.17 => 6.1.17
    @tamagui/babel-plugin: ^1.108.3 => 1.108.3
    @tamagui/config: ^1.108.3 => 1.108.3
    @tamagui/lucide-icons: ^1.108.3 => 1.108.3
    @tamagui/metro-plugin: 1.108.2 => 1.108.2
    @tamagui/toast: ^1.108.3 => 1.108.3
    @types/react: ~18.2.79 => 18.2.79
    @types/react-native: ^0.73.0 => 0.73.0
    babel-preset-expo: ^11.0.6 => 11.0.6
    burnt: ^0.12.2 => 0.12.2
    expo: ~51.0.9 => 51.0.9
    expo-font: ~12.0.6 => 12.0.6
    expo-linking: ~6.3.1 => 6.3.1
    expo-router: ~3.5.15 => 3.5.15
    expo-splash-screen: ~0.27.4 => 0.27.4
    expo-status-bar: ^1.12.1 => 1.12.1
    expo-system-ui: ~3.0.4 => 3.0.4
    expo-web-browser: ~13.0.3 => 13.0.3
    react: 18.2.0 => 18.2.0
    react-dom: 18.2.0 => 18.2.0
    react-native: 0.74.1 => 0.74.1
    react-native-reanimated: ~3.10.1 => 3.10.1
    react-native-safe-area-context: 4.10.1 => 4.10.1
    react-native-screens: ~3.31.1 => 3.31.1
    react-native-svg: 15.2.0 => 15.2.0
    react-native-web: ^0.19.12 => 0.19.12
    tamagui: ^1.108.3 => 1.108.3
    typescript: ^5.5.2 => 5.5.2
```

**Comment by jonsherrard** [comment]
Will get a fix in for this, but in the meantime run yarn tamagui:upgrade and things should start working.

**Comment by jonsherrard** [comment]
Fixed in #2851
